README.md

```diff
@@ -1,29 +1,39 @@
 ## PySTAC
-![Build Status](https://github.com/stac-utils/pystac/workflows/CI/badge.svg?branch=develop)
+![Build Status](https://github.com/stac-utils/pystac/workflows/CI/badge.svg?branch=main)
 [![PyPI version](https://badge.fury.io/py/pystac.svg)](https://badge.fury.io/py/pystac)
 [![Documentation](https://readthedocs.org/projects/pystac/badge/?version=latest)](https://pystac.readthedocs.io/en/latest/)
 [![codecov](https://codecov.io/gh/stac-utils/pystac/branch/main/graph/badge.svg)](https://codecov.io/gh/stac-utils/pystac)
-[![Gitter chat](https://badges.gitter.im/azavea/pystac.svg)](https://gitter.im/azavea/pystac)
+
```

Comment:
Perhaps add the STAC python channel as a resource, either as a badge or else where in the readme, as that's where a lot of PySTAC related conversation happens

```suggestion
[![Gitter](https://badges.gitter.im/SpatioTemporal-Asset-Catalog/python.svg)](https://gitter.im/SpatioTemporal-Asset-Catalog/python?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
```